lib/guides_style_18f/includes/scripts.html

```diff
@@ -1,6 +1,6 @@
 <script src="{% guides_style_18f_asset_root %}/assets/js/vendor/jquery-1.11.2.min.js"></script>
 <script src="{% guides_style_18f_asset_root %}/assets/js/accordion.js"></script>
-<script src="{% guides_style_18f_asset_root %}/assets/js/vendor/private-eye-1.0.0.js"></script>
+<script src="{% guides_style_18f_asset_root %}/assets/js/vendor/private-eye-1.1.0.js"></script>
```

Comment:
Ha...this broke existing sites. Once more in favor of https://github.com/18F/pages-server/issues/55.